WooCommerce/WooCommerceTests/POS/Mocks/MockPOSOrderService.swift

```diff
@@ -19,7 +19,9 @@ class MockPOSOrderService: POSOrderServiceProtocol {
         return order
     }
 
-    func sendReceipt(order: Yosemite.Order, recipientEmail: String) async throws { }
+    func updatePOSOrder(order: Order, recipientEmail: String) async throws {
+        // no-op
```

Comment:
We could consider updating PointOfSaleOrderController tests to confirm that both update and sendReceipt are called 👍

Reply:
Good point, added: https://github.com/woocommerce/woocommerce-ios/pull/14663/commits/51cab4d2ee219a5a8c37fc4e7681d6736c113c97